### DISCUSSION TITLE
Noob Level : Failed SO-Test

### DISCUSSION THREAD

**Discussion by iriwsky** [Security-Onion-Solutions/securityonion#2943] [Unsupported Versions] [answered]
I tried to run so-test after installing.  My environment is vmware workstation pro 16, SO version 2.3.21.  How can I fix this?  Is there a script that I need to run to create 'samples and so-tcpreplay?  so-status are all green.  Thank you!
![Security-Onion-so-test](https://user-images.githubusercontent.com/78871624/107703914-9ca78900-6c79-11eb-97ce-ae3ab51eb82c.png)

**Comment by cam98mac** [comment, marked as answer]
I had this same problem with working through the training. Unfortunately the so-test command in version 2.3.21 doesn't work. 

It is being worked on and it looks like it will be fixed in the next release. It is being tracked in comment #2463 .

**Reply by iriwsky** [reply]
Thank you for the info!  Closing the discussion.
